File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        map<char,int> a,b;
        int n=s.size();
        int m=t.size();
        if (n !=m) return false;
        for (int i=0;i<n;i++){
            a[s[i]]++;
            b[t[i]]++;
        }
        return a==b;
        
    }
};

File: Data Structures & Algorithms/non-overlapping-intervals/submission-0.py
class Solution:
    def eraseOverlapIntervals(self, intervals: List[List[int]]) -> int:
        intervals.sort(key=lambda x:x[1])
        cnt = 0
        lim = intervals[0][1]
        for i in range(1,len(intervals)):
            if intervals[i][0]<lim:
                cnt+=1
            else:
                lim=intervals[i][1] 
        return cnt